src/GraphControl/GraphControl.vcxproj.filters

```diff
@@ -68,4 +74,8 @@
   <ItemGroup>
     <None Include="$(ProjectDir)\GraphingImplOverrides.props" />
   </ItemGroup>
+  <ItemGroup>
+    <CopyFileToFolders Include="$(GraphingImplDll)" />
```

Comment:
I think you can remove this

Reply:
VS Keeps adding those for some reason.  Removed.